azure/packages/azure-local-service/package-lock.json

```diff
@@ -0,0 +1,7680 @@
+{
```

Comment:
We don't want to add this file. The azure release group has a single set of lock files at its root. You'll want to re-run `npm install` from the root of the release group, and remove this file.

Reply:
This file's addition is also why the PR pipeline is currently failing (https://dev.azure.com/fluidframework/public/_build/results?buildId=108376&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=baa7d181-d9ee-513a-8016-f44a98fd040e)